docs/02-analyze/01-duration-terms.adoc

```diff
@@ -1,7 +1,7 @@
 // tag::DE[]
 [width=50%]
 |===
-| Dauer: 210 Min. | Übungszeit: 0 Min.
```

Comment:
Es wäre schön, wenn wir die Begriffe bitte behalten können, damit die auch in allen Lehrplänen gleich sind. Gerade die englischen Begriffe wurden von einem Büro übersetzt und so vorgegeben.

Reply:
Ist "Dauer" die Gesamtdauer der Sektion inkl. Übungszeit oder nur die Dauer des Theorieblocks ohne Übungszeit? Diese Unklarheit war u. a. der Grund für die Umbenennung.

Reply:
Nur Theorie.